bot/exts/help_channels/_cog.py

```diff
@@ -535,3 +539,76 @@ async def update_available_help_channels(self) -> None:
         )
         self.dynamic_message = new_dynamic_message["id"]
         await _caches.dynamic_message.set("message_id", self.dynamic_message)
+
+    @staticmethod
+    def _serialise_session_participants(participants: set[int]) -> str:
+        """Convert a set to a comma separated string."""
+        return ','.join(str(p) for p in participants)
+
+    @staticmethod
+    def _deserialise_session_participants(s: str) -> set[int]:
+        """Convert a comma separated string into a set."""
+        return set(int(user_id) for user_id in s.split(",") if user_id != "")
```

Comment:
Aren't sets in redis more appropriate for this sort of thing? It would remove the need to serialise and deserialise. It wouldn't even need to retrieve all items since there are commands to check set membership.

Reply:
Async Rediscache doesn't support sets.

Reply:
aioredis does support it and async-redis cache can trivially be extended to support it (yes I think it's worth doing that).

Reply:
Note I'm not suggesting implementing the set operations with all the type conversion stuff. Just a basic SADD call or whatever that will store a string as given.

Reply:
I'm not too familiar with Redis can you please give more information about how I would implement this? Also I think it would be best to do this as a separate feature.

Reply:
That's alright. Let's just leave your implementation for now. I can try to merge set support into async-rediscache in the future.

Reply:
Alright, sounds good!